python/python/res/server/simulation_context.py

```diff
@@ -93,3 +96,10 @@ def __repr__(self):
         fmt = '%s, #running = %d, #success = %d, #failed = %d, #waiting = %d'
         fmt =  fmt % (running, numRunn, numSucc, numFail, numWait)
         return 'SimulationContext(%s)' % fmt
+
+    def get_result_fs(self):
```

Comment:
To me it seems like `result_fs`and `run_context` are good candidates for being properties..